bot/__init__.py

```diff
@@ -0,0 +1,6 @@
+try:
+    from dotenv import load_dotenv
+    print("Found .env file, loading environment variables from it.")
```

Comment:
Shouldn't this use logging?

Reply:
Fixed with [15852ba](https://github.com/python-discord/sir-robin/pull/10/commits/15852ba5c2b9e27bf94125246b6c83e21a4a3a8c)